visual-inspection/inspection-backend/inspection/api.py

```diff
@@ -19,6 +19,8 @@ def predict_weather(file: UploadFile = File(...)) -> Prediction:
 # TODO: Allow non-nested settings
 class ExplanationRequest(BaseModel):
     method: str = _settings.default_explainer
+    index_of_label_to_explain: int = 0
+    positive_only_parameter: bool = False
```

Comment:
Settings should be provided as `settings` to not clutter the request input and maintain consistency across use case and explanation methods.